_includes/code/blackjack.cc

```diff
@@ -0,0 +1,69 @@
+#include <queue>
+
+
+using namespace std;
+
+// Suits
+int spades = 0;
+int hearts = 1;
+int diamonds = 2;
+int clubs = 3;
```

Comment:
How do you feel about making this an enum class?

```C++
enum class Suit {
  kSpades,
  kHearts,
  kDiamonds,
  kClubs,
};
```

That gives you some type safety.